docs/events/timeline-ui-overview.asciidoc

```diff
@@ -12,9 +12,15 @@ by clicking *+ Add field*.
 [role="screenshot"]
 image::images/timeline-ui-updated.png[example Timeline with several events]
 
+<<<<<<< HEAD
 Timelines are responsive, and they persist as you move through the {security-app}
 collecting data. Auto-saving ensures that the results of your investigation are
 available for later review. To record and share your findings with others, attach your Timeline to a <<cases-overview, case>>.
+=======
+To avoid losing your changes, you must save the Timeline before moving to a different {security-app} page.
+
+If you change an existing Timeline, you can use the **Save as new timeline** toggle to make a new copy of the Timeline, without overwriting the original one.
+>>>>>>> 7122ae66 (Save timeline as new (#4437))
```

Comment:
```suggestion
To avoid losing your changes, you must save the Timeline before moving to a different {security-app} page.

If you change an existing Timeline, you can use the **Save as new timeline** toggle to make a new copy of the Timeline, without overwriting the original one.
```